### PR TITLE
Fixed issues related to flows and atoms initialization

### PR DESCRIPTION
**Pull request by shtripat**
Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

File: tendrl/commons/jobs/__init__.py [hunk 1]
```diff
@@ -94,7 +94,9 @@ def stop(self):
     def invoke_flow(self, flow_fqn, job):
         # flow_fqn eg:tendrl.node_agent.objects.abc.flows.temp_flows
         if "tendrl" in flow_fqn and "objects" in flow_fqn:
-            obj_name, flow_name = flow_fqn.split(".objects.")[-1].split(
+            obj_name = flow_fqn.split(".objects.")[-1].split(
+                ".flows.")[-2]
```
Comment (r0h4n) [inline tendrl/commons/jobs/__init__.py]: the older code was doing the same, whats the difference?

Comment (shtripat) [inline tendrl/commons/jobs/__init__.py]: Earlier we were trying to get obj_name and flow_name both in the same call. The value of obj_name used to be correct but flow name was not. Flow name used toe be reported as e.g. `create_volume.CreateVolume` whereas we need only `CreateVolume`, so we get these values separately now. Thats all...

File: tendrl/commons/objects/atoms/__init__.py [hunk 2]
```diff
@@ -19,11 +19,11 @@ def __init__(
                                              self.obj.__name__)
         atom_def = obj_def.atoms[self.__class__.__name__]
 
-        self.inputs = inputs or atom_def['inputs']['mandatory']
-        self.enabled = enabled or atom_def['enabled']
-        self.outputs = outputs or atom_def['outputs']
-        self.uuid = uuid or atom_def['uuid']
-        self.help = help or atom_def['help']
+        self.inputs = inputs or atom_def.get('inputs')['mandatory']
```
Comment (r0h4n) [inline tendrl/commons/objects/atoms/__init__.py]: atom_def.get('inputs')['mandatory'] should be atom_def.get('inputs').get("mandatory")

Comment (shtripat) [inline tendrl/commons/objects/atoms/__init__.py]: Done

File: tendrl/commons/jobs/__init__.py [hunk 3]
```diff
@@ -94,7 +94,9 @@ def stop(self):
...
+            flow_name = flow_fqn.split(".objects.")[-1].split(
```
Comment (r0h4n) [inline tendrl/commons/jobs/__init__.py]: flow_name = flow_fqn.split(".flows.")[-1].split(".")[-1]

File: tendrl/commons/jobs/__init__.py (hunk 1 above)
Comment (r0h4n) [inline tendrl/commons/jobs/__init__.py]: obj_name = flow_fqn.split(".objects.")[-1].split(".")[0]

much cleaner

Comment (shtripat) [inline tendrl/commons/jobs/__init__.py]: Done

File: tendrl/commons/jobs/__init__.py (hunk 3 above)
Comment (shtripat) [inline tendrl/commons/jobs/__init__.py]: Done

**Comment by shtripat** [comment]
Verified with latest create/delete pool changes and works as expected.

**Pull request merged**